src/libmongoc/tests/test-mongoc-cluster.c

```diff
@@ -1746,7 +1753,7 @@ _test_cmd_on_unknown_serverid (bool pooled)
    uri = test_framework_get_uri ();
    /* Set a high heartbeatFrequencyMS so subsequent topology scans do not
     * interfere with the test. */
-   mongoc_uri_set_option_as_int32 (uri, MONGOC_URI_HEARTBEATFREQUENCYMS, 99999);
+   mongoc_uri_set_option_as_int32 (uri, MONGOC_URI_HEARTBEATFREQUENCYMS, 5000);
```

Comment:
Does the comment above need to be changed? The default is 10000 for pooled and 60000 for single-threaded, so this is no longer a "high heartbeatFrequencyMS". What was the reason for lowering this vs. removing it entirely?

Reply:
I updated the comment. This test was written before streamable hello. Marking the server as unknown manually ends up causing this test to take the full heartbeatFrequencyMS to rediscover a server to send endSessions.